public/src/components/group/GroupForm.tsx

```diff
@@ -77,7 +79,7 @@ const GroupForm = (): JSX.Element | null => {
             return (
                 <li hidden={search(enrollment)} key={id.toString()} className="list-group-item">
                     {enrollment.user?.Name}
-                    <i className="badge-pill badge-success ml-2 clickable float-right" onClick={() => actions.updateGroupUsers(enrollment.user)}>+</i>
+                    <Button classname="ml-2 float-right" onclick={() => actions.updateGroupUsers(enrollment.user)} type={ButtonType.BADGE} color={Color.GREEN} text={"+"} />
```

Comment:
make more readable by using multiple lines (one for each field) ... same for Line 94 below.